cmd/internal/pipeline/evaluation_target_test.go

```diff
@@ -0,0 +1,83 @@
+package pipeline
+
+import (
+	"fmt"
+	"github.com/hacbs-contract/ec-cli/cmd/internal/utils"
+	"github.com/spf13/afero"
+	"os"
+	"testing"
+)
+
+var pipelineDefinitionDir = "./test_data/pipeline_definitions"
+
+func deployPipelineFile() error {
```

Comment:
I _think_ we could replace this with `//go:embed` something like

```go
//go:embed test_data/pipeline_definitions/
var pipelineDefinitions embed.FS
```

And then in tests we could do something like:

```go
defaultAppFS := util.AppFs
util.AppFs = afero.FromIOFS{fs: pipelineDefinitions}
defer util.AppFs = defaultAppFS
```